Jasmine/Grid/ViewModels/GridViewModel.swift

```diff
@@ -2,32 +2,43 @@ import Foundation
 
 class GridViewModel: GridViewModelProtocol {
     /// Stores the grid data that will be used to display in the view controller.
-    private(set) var gridData: [Coordinate : String] = [:]
+    private(set) var gridData: [Coordinate: String] = [:]
```

Comment:
I'll use gridData to favor brevity at the moment, but do note that:

1. when filling in gridData for the first time, updateGridData would need to be called 16 times
2. the built-in swap function cannot be used since it will introduce concurrent update

If updateGridData is expensive I will revert back.